templates/etc/audit/rules.d/hardening.rules.j2

```diff
@@ -1,5 +1,4 @@
 # {{ ansible_managed }}
-# https://github.com/konstruktoid/ansible-role-hardening
```

Comment:
Why was this removed?

Reply:
This was backported from upstream commit: https://github.com/cisagov/ansible-role-hardening-2/commit/b3571d307ab670304eba4e9cf3886e1bda99a3db

Reply:
OK, but please see [this comment](https://github.com/cisagov/ansible-role-hardening-2/pull/1#discussion_r1388217368), as it also applies here.